SECURITY.md

```diff
@@ -0,0 +1,24 @@
+# Security Policy
+
+## Supported Versions
+
+Security updates are applied to the latest MINOR version of Jekyll, and the version used by GitHub Pages, v3.9.x.
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 4.2.x   | :white_check_mark: |
+| 3.9.x   | :white_check_mark: |
+| < 3.8.x | :x:                |
+
+## Reporting a Vulnerability
+
+Please report vulnerabilities by sending an email to security@jekyllrb.com with the following information:
+
+1. A description of the vulnerability
+1. Reproduction steps and/or a sample site (share a private repo to the Jekyll Core Team and @parkr)
+2. Your contact information
+
+The Jekyll security team will respond to your submission and notify you whether it has been confirmed by the team. 
+Your confidentiality is kindly requested as we work on a fix. We will provide our patch to you to test and verify that the vulnerability has been closed.
+Once a fix is verified, we will release PATCH versions of the supported MINOR versions and assign a CVE to the vulnerability. You will receive credit in our release post.
```

Comment:
Perhaps some bounty (w.r.t exploit severity) in addition to credit will incentivize the security-bug-hunt..?

Reply:
I'm not sure we have funds to guarantee a bounty (and paying becomes a bit more complicated), so the best thing I think we can consistently offer at this stage is The Glory of Being on Our Website In Perpetuity.

Reply:
Yeah, I don't think the consistently have the funds to do this and I'd hate to be like "you get money! (until the money runs out)". I like the idea though.